data/json/dreams.json

```diff
@@ -612,6 +640,8 @@
     "type": "dream",
     "messages": [
       "You excitedly web up an interloper and prepare to feast… nope, dream.",
+      "You have a nightmare about being washed out of a drainpipe.",
```

Comment:
Nursery rhyme references are a bit much.
```suggestion
```

Reply:
Aw, I liked this one